docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+services:
+  server:
+    image: go-fast-cdn
```

Comment:
Leaving this as a note, but if we decide to go the official image route we will need to change this